internal/backends/sqlite/collection.go

```diff
@@ -76,6 +112,19 @@ func (c *collection) Query(ctx context.Context, params *backends.QueryParams) (*
 	}, nil
 }
 
+// eq returns filter and boolean indicating if filter for `v` is supported.
+func eq(v any) (string, bool) {
+	switch v := v.(type) {
+	case *types.Document, *types.Array, float64, string, types.Binary, bool, time.Time,
+		types.NullType, types.Regex, int32, types.Timestamp, int64:
+		return "", false
+	case types.ObjectID:
+		return `_jsonb->? @> ?`, true
```

Comment:
In SQLite, that would be `fmt.Sprintf("%s = ?", metadata.IDColumn)`. See Update and Delete for examples